framework/components/ema-status/ema-status.config.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  title: 'Status',
+  label: 'Status',
+  status: 'ready',
+  tags: ['molecule'],
+  collated: true,
+  collator(markup, item) {
+    return `<!-- Start: @${item.handle} -->\n
+      ${markup}\n
+      <!-- End: @${item.handle} -->\n`;
+  },
+  default: 'info',
+  variants: [
+    {
+      name: 'info',
+      label: 'Info',
+      context: {
+        extra_class: 'ema-status--info',
```

Comment:
This class should be handled in twig template.
We generally use a context variable `variant` or `mixin` to handle it